assets/js/custom/SnackbarNotification.js

```diff
@@ -0,0 +1,12 @@
+import { showSnackbar } from '../components/snackbar'
+
+export class SnackbarNotification {
```

Comment:
`js/custom/SnackbarNotification` is a bit too generic. Can it be more specific like the other classes in this commit?
Also `Notification` is a bit redundant